README.md

```diff
@@ -1,14 +1,20 @@
 # Project
 
-> This repo has been populated by an initial template to help get you started. Please
-> make sure to update the content to build a great experience for community-building.
+This utility can be used to detect the supported TLS version for a given CosmosDB account.
 
-As the maintainer of this project, please make a few updates:
+Note that this utility would only be able to test the account against TLS versions supported by the client system running the script.
 
-- Improving this README.MD file to provide a great experience
-- Updating SUPPORT.MD with content about this project's support experience
-- Understanding the security reporting process in SECURITY.MD
-- Remove this section from the README
+Supported TLS version:
+- TLS V1
+- TLS V1.1
+- TLS V1.2
+- TLS V1.3
+
+Run the utility with a given CosmosDB endpoint as follows
```

Comment:
If these are all the options, this is probably fine for starting out. Ideally, you'd have documentation on what are the available options and how to understand the output from the tool.

Reply:
These are actually all the TLS version the tool checks against. I've updated Readme with how to use and interpret results.